src/main/java/org/assertj/core/util/DoubleComparator.java

```diff
@@ -23,7 +21,7 @@ public DoubleComparator(double epsilon) {
   }
 
   @Override
-  public int compare(Double x, Double y) {
+  protected int compareNonNull(Double x, Double y) {
```

Comment:
I am a little bit curious what is unused `private` static method `complexCloseEnough()` present for?